lib/geolocation/simulator.js

```diff
@@ -14,27 +14,33 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+'use strict';
+
 var console = require('console')
 
 var Simulator = function () {
-   throw("TODO: unimplemented");
-}
+  this.value = {
+    latitude: 0,
+    longitude: 0
+  };
+};
 
 Simulator.prototype.read = function (callback) {
-  throw("TODO: unimplemented");
-  if (callback) return callback(null, this.value)
+  this.value.latitude = Math.random() * 90;
+  this.value.longitude = Math.random() * 360 - 180;
```

Comment:
mistake may you can commit style cleanup only patches just after this one is in

Reply:
I figured I'd just clean up the original commits, as there wasn't too much info in the mistakes...